st2common/st2common/openapi.yaml.j2

```diff
@@ -5380,6 +5388,14 @@ definitions:
         items:
           type: string
     additionalProperties: false
+  ActionDeleteSchema:
+    type: object
+    properties:
+      remove_files:
+        type: boolean
+        description: Force delete action files from disk
```

Comment:
Minor comment - should this be reworded as in ohter schema, as its not really a force - its just a flag.